lib/ChunklistManifestGenerator.js

```diff
@@ -98,12 +98,14 @@ module.exports = function(outputFolderPath, filename, dvrWindowSize){
             currentManifest.set('mediaSequence', currentSequenceNumber + numberOfRemovedItems);
 
             // Update manifest media sequence duration with maximal #EXTINF (rounded up)
-            var manifestLongestChunk = _.max(currentManifest.items.PlaylistItem, function(item) {
-                return item.get('duration');
-            });
 
-            currentManifest.set('targetDuration', Math.ceil(manifestLongestChunk.get('duration')));
+            var manifestLongestChunkDuration = _.max(_.map(currentManifest.items.PlaylistItem, function(item) {
```

Comment:
isn't the map function redundant?
_.max(currentManifest.items.PlaylistItem, function(item){